index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- end of google fonts -->
+    <!-- fontawesome -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <!-- end of fontawesome -->
+    <!-- style css -->
+    <link rel="stylesheet" href="style.css" />
+    <!-- end of style css -->
+    <title>portf</title>
+  </head>
+  <body>
+    <div class="header">
+      <div class="nav">
+        <div class="logo">
+          <a href="index.html"><h1 class="logo">LOGO</h1></a>
+        </div>
+
+        <div class="menu-bar">
+          <i class="fa-solid fa-bars"></i>
+        </div>
```

Comment:
- [ ] Please make sure to use the appropriate semantics for your pages as this ensures that search engines and other accessibility software like screen readers can easily identify the elements on your page. 🎯
    - The code from lines `31-38` should be in a self-contained `header` element and please make sure to put your hamburger icon inside a `nav` element as well as this ensures it's part of a navigation element.
- [ ] Also please make sure that all your page elements are visible for a screen width of at least 375px (mobile) as your menu Icon seems to be overflowing,
    ![firefox_iv4CLuTi0o](https://user-images.githubusercontent.com/104968236/227497250-c62b3858-38c9-4194-9014-872d2233eddd.png)
    - **HINT:** I believe the source of this problem is the margin you've added on the `.header` class kindly replace that with padding and add the appropriate spacing.